wallet/node/Main.hs

```diff
@@ -80,7 +80,6 @@ actionWithWallet sscParams nodeParams ntpConfig wArgs@WalletArgs {..} = do
                 runWRealMode
                     db
                     conn
-                    (AddrCIdHashes ref)
                     syncRequestsQueue
```

Comment:
Very weird that GitHub is pulling this change in the diff, as this is coming from a commit that @nc6 merged a while ago in `develop`. Go figure 🤔

Reply:
The branch was very old and I had to redo some of that work as part of merge conflicts